node/dataSources/subscription.ts

```diff
@@ -10,21 +9,35 @@ interface AggregationsArgs {
   interval: string
 }
 
-export class SubscriptionDataSource extends RESTDataSource<ServiceContext> {
-  constructor() {
-    super()
-  }
+interface SubscriptionsOrdersArgs {
```

Comment:
I think `Subscription` instead `Subscriptions` it's better.

Reply:
👍